processor/src/main/java/org/mapstruct/ap/internal/model/source/SubclassMappingOptions.java

```diff
@@ -34,14 +36,16 @@ public class SubclassMappingOptions extends DelegatingOptions {
     private final TypeMirror target;
     private final TypeUtils typeUtils;
     private final SelectionParameters selectionParameters;
+    private final SubclassMappingGem subclassMappingGem;
```

Comment:
Let's name this `subclassMapping`. We are not adding the `Gem` prefix to fields and variables. e.g. have a look at `MappingOptions`